references/classification/utils.py

```diff
@@ -181,7 +181,7 @@ def update_parameters(self, model):
             else:
                 p_swa.detach().copy_(self.avg_fn(p_swa.detach(), p_model_,
                                      self.n_averaged.to(device)))
-            self.n_averaged += 1
+        self.n_averaged += 1
```

Comment:
Maybe in a separate PR?
If [#65495](https://github.com/pytorch/pytorch/pull/65755) is merged first, can we remove this custom function?

Reply:
Yeap, that's exactly the plan. Either a separate bug fixing PR or a complete removal of this util and adoption of the PyTorch's core class that @prabhat00155 wrote.

I'll wait to see which one comes first. :)